S2VX.Game/Editor/EditorScreen.cs

```diff
@@ -98,100 +99,116 @@ public class EditorScreen : Screen {
             Seek(editorSettings.TrackTime);
             PlaybackSetRate(editorSettings.TrackPlaybackRate);
             SnapDivisor = editorSettings.SnapDivisor;
             NotesTimeline.DivisorIndex = editorSettings.BeatSnapDivisorIndex;
+        }
 
-            // Sets the same clock for sections dependent on the Track
-            var trackClock = new FramedClock(Track);
-            Story.Clock = trackClock;
-            NotesTimeline.Clock = trackClock;
-            Timeline.Clock = trackClock;
-            ToolContainer.Clock = trackClock;
-
-            RelativeSizeAxes = Axes.Both;
-            Size = Vector2.One;
+        private EditorUI CreateEditorUI() {
+            var editorUI = new EditorUI {
+                RelativeSizeAxes = Axes.Both,
+                Children = new Drawable[] {
+                        NoteSelectionIndicators,
+                        ToolContainer,
+                        NotesTimeline,
+                        new EditorInfoBar(),
+                        CreateMenu(),
+                        Timeline,
+                        CommandPanel
+                    }
+            };
+            editorUI.State.Value = Visibility.Visible;
+            return editorUI;
+        }
 
-            ToolContainer.Child = ToolState;
-            InternalChildren = new Drawable[]
-            {
-                Track,
-                Story,
-                NoteSelectionIndicators,
-                ToolContainer,
-                NotesTimeline,
-                new EditorInfoBar(),
-                new BasicMenu(Direction.Horizontal, true)
+        private BasicMenu CreateMenu() =>
+            new BasicMenu(Direction.Horizontal, true) {
+                BackgroundColour = Color4.Black.Opacity(0.9f),
+                Width = 1,
+                RelativeSizeAxes = Axes.X,
+                Items = new[]
                 {
-                    BackgroundColour = Color4.Black.Opacity(0.9f),
-                    Width = 1,
-                    RelativeSizeAxes = Axes.X,
-                    Items = new[]
+                    new MenuItem("Project")
                     {
-                        new MenuItem("Project")
+                        Items = new[]
                         {
-                            Items = new[]
-                            {
-                                new MenuItem("Refresh (Ctrl+R)", ProjectRefresh),
-                                new MenuItem("Save (Ctrl+S)", ProjectSave),
-                                new MenuItem("Play (G)", ProjectPlay),
-                                new MenuItem("Quit (Esc)", ProjectQuit),
-                            }
-                        },
-                        new MenuItem("Edit")
+                            new MenuItem("Refresh (Ctrl+R)", ProjectRefresh),
+                            new MenuItem("Save (Ctrl+S)", ProjectSave),
+                            new MenuItem("Play (G)", ProjectPlay),
```

Comment:
Should we rename this Preview Gameplay to avoid confusion with Play/Pause?

Reply:
Agreed